anima/ui/user_dialog.py

```diff
@@ -0,0 +1,167 @@
+from PySide2 import QtWidgets
+from PySide2 import QtCore
+from shiboken2 import wrapInstance
+
+from anima.ui.base import AnimaDialogBase, ui_caller
+
+import maya.OpenMayaUI as omui
+
+
+
+def UI(app_in=None, executor=None, **kwargs):
+    """
+    :param app_in: A Qt Application instance, which you can pass to let the UI
+      be attached to the given applications event process.
+
+    :param executor: Instead of calling app.exec_ the UI will call this given
+      function. It also passes the created app instance to this executor.
+
+    """
+    user_dialog=UserDialog()
+    try:
+        user_dialog.close()
+        user.dialog.deleteLater()
+    except:
+        pass
+
+    return ui_caller(app_in, executor, UserDialog, **kwargs)
+    
+
+    
+def maya_main_window():
+    """
+    Return the Maya main window widget as a Python object
+    """
+    
+    main_window_ptr=omui.MQtUtil.mainWindow()
+    return wrapInstance(long(main_window_ptr),QtWidgets.QWidget)
+    
+
+class UserDialog(QtWidgets.QDialog):
+    """
+    The User Creation Dialog
+    """
+    def __init__(self,parent=maya_main_window()):
+        super(UserDialog,self).__init__(parent)
+        
+        self.setWindowTitle("User Dialog")
+        self.setWindowFlags(self.windowFlags()^QtCore.Qt.WindowContextHelpButtonHint)
+        
+        
+        self.create_widgets()
+        self.create_layouts()
+        self.create_connections()
+        
+    def create_widgets(self):
+        
```

Comment:
Please create a docstring for every methods you've created. It could be as simple as "Creates widgets".